<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="playlist">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>331</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="playPushButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>330</y>
      <width>88</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pausePushButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>330</y>
      <width>88</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
   <widget class="QPushButton" name="volUpPushButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>400</y>
      <width>88</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Vol +</string>
    </property>
   </widget>
   <widget class="QPushButton" name="volDwnPushButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>400</y>
      <width>88</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Vol -</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addPushButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>20</y>
      <width>88</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removePushButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>70</y>
      <width>88</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopPushButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>330</y>
      <width>88</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QLabel" name="songDurationLabel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>300</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
